include/utils/string.hpp

```diff
@@ -97,7 +97,8 @@ namespace string_util {
   string floating_point(double value, size_t precision, bool fixed = false, const string& locale = "");
   string filesize_mib(unsigned long long kibibytes, size_t precision = 0, const string& locale = "");
   string filesize_gib(unsigned long long kibibytes, size_t precision = 0, const string& locale = "");
-  string filesize_gib_mib(unsigned long long kibibytes, size_t precision_mib = 0, size_t precision_gib = 0, const string& locale = "");
```

Comment:
I also ran clang-format but restored this change back as I didn't touch this part of the code

Reply:
At some point we will have a CI job that tests for formatting and we will always have all files correctly formatted so this doesn't happen anymore.